analytics.py

```diff
@@ -31,7 +32,21 @@ def plot_model_distribution():
     plt.pie(models.values(), labels=models.keys(), autopct='%1.1f%%')
     plt.title("Model Usage Distribution")
     plt.savefig("model_distribution.png")
+    plt.cla()
 
+def plot_response_time_distribution():
+    data = pd.read_json(LOG_FILE, lines=True)
+    dates = data['timestamp']
+    response_times = data['response_time']
+    
+    plt.plot(dates, response_times)
+    plt.xticks(rotation=45)
+    plt.ylabel("Response Time (s)")
+    plt.xlabel("Timestamp")
+    plt.title("Response Time Over Time")
+    plt.tight_layout()
+    plt.savefig("response_time.png")
+    plt.cla()
```

Comment:
Added plt.cla to reset graphs, otherwise line graphs get overlayed on each other. It wasn't a problem before because there was only 1 line graph in plot_token_usage().